Libraries/Text/RCTTextFieldManager.m

```diff
@@ -39,8 +39,16 @@ - (BOOL)textField:(RCTTextField *)textField shouldChangeCharactersInRange:(NSRan
   }
 
   if (textField.maxLength == nil || [string isEqualToString:@"\n"]) {  // Make sure forms can be submitted via return
+    if (textField.autocorrectionType == UITextAutocorrectionTypeYes) {
```

Comment:
What about `UITextAutocorrectionTypeDefault`, do we need additional handling for that case?